src/app/_component/home-page.tsx

```diff
@@ -1,9 +1,13 @@
 import Banner from '@/components/banner/banner';
+import Footer from '@/components/footer/footer';
+import Header from '@/components/header/header';
 
 export function HomePage() {
   return (
     <div>
+      <Header />
```

Comment:
This was my bad,

We don't need to move the header and Footer to each page anymore. We plan to use client-side rendering for Headers.